src/store/index.js

```diff
@@ -55,6 +62,16 @@ export default new Vuex.Store({
 				folders: [],
 				envelopeLists: {},
 			},
+			[PRIORITY_INBOX_UID]: {
+				id: PRIORITY_INBOX_ID,
+				accountId: 0,
```

Comment:
Might be out of the scope of this PR, but why are there both an `account.id` and `account.accountId`? And why can they be different?

Reply:
Someone once had the idea to use `accountId` instead of `id`, while now we have both. To be honest I don't know why that is, but we should slowly move towards just `id` :)

Reply:
Hmm, then should these not be different here? Assuming this difference is important for L205 and L302? Urgh, I hate asking questions about other people's code 😖

Reply:
> I hate asking questions about other people's code confounded

No, please do :)

I think we should unify accountId and id and use it consistently.

> Hmm, then should these not be different here?

No, this new inbox is also part of the unified account. Though we might reconsider the name *Unified account* now that it has more than one purpose.